### DISCUSSION TITLE
Module -V how to hide mesh edges in the results

### DISCUSSION THREAD

**Discussion by xshang93** [neperfepx/neper#290] [Q&A] [answered]
I'm trying to generate the results without the mesh edges, i.e., keep only the stress distribution but not the black edges (See in picture attached). The **-meshshow 0** seems to hide the results. **-showedge none** only works on the edges of tessellations, not on the results.
![1_1p_s11_deform](https://user-images.githubusercontent.com/86078353/132291757-9ba07408-76eb-45be-872a-eba14c02676b.png)

TIA

**Comment by rquey** [comment, marked as answer]
The thick lines are 1D elements and are controlled by `-showelt1d` and `-dataelt1drad`. The thin lines are 2D element edges and are controlled by and `-dataelt2dedgerad` (can be set to 0).

**Reply by xshang93** [reply]
Awesome! Thanks so much. This is actually in 3d and I'm viewing it from the front. I set the `-dataelt3dedgerad` to zero and all edges (thick and thin) disappeared. I think that's due to both the grains and elements are in 3D?

By the way, this must be something I missed from the manual, but could you advise how I can extrapolate the results and get a smooth stress distribution, rather than discrete colors in each mesh?

Thank you!

**Reply by rquey** [reply]
> Awesome! Thanks so much. This is actually in 3d and I'm viewing it from the front. I set the `-dataelt3dedgerad` to zero and all edges (thick and thin) disappeared. I think that's due to both the grains and elements are in 3D?

This shouldn't be.  Thick lines are from the 1D elements. So, make sure to use `-showelt1d all` and to set `-dataelt1drad`.

> By the way, this must be something I missed from the manual, but could you advise how I can extrapolate the results and get a smooth stress distribution, rather than discrete colors in each mesh?

We can interpolate node data within elements (which creates a smooth distribution), but we don't generate smooth distributions from element data. It will probably come...

**Reply by xshang93** [reply]
> This shouldn't be. Thick lines are from the 1D elements. So, make sure to use `-showelt1d all` and to set `-dataelt1drad`.

Makes sense. I tried this and it's exactly as you said. 

> We can interpolate node data within elements (which creates a smooth distribution), but we don't generate smooth distributions from element data. It will probably come...

Okay thanks for clarifying. I guess I'll just refine my mesh and hope to see a smooth enough results
